ietf/doc/utils.py

```diff
@@ -1317,3 +1318,33 @@ def fuzzy_find_documents(name, rev=None):
 
     FoundDocuments = namedtuple('FoundDocuments', 'documents matched_name matched_rev')
     return FoundDocuments(docs, name, rev)
+
+def bibxml_for_draft(doc, rev=None):
+
+    if rev != None and rev != doc.rev:
```

Comment:
Since the refactor makes this non-obvious, the primary fix is on this line at `and rev != doc.rev`